presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -5719,7 +5719,7 @@ public void testShowTablesFrom()
     @Test
     public void testShowTablesLike()
     {
-        MaterializedResult result = computeActual("SHOW TABLES LIKE 'or%'");
+        MaterializedResult result = computeActual("SHOW TABLES LIKE 'orde%'");
```

Comment:
maybe change assertion to not assume just `orders` is listed. But allow any number of tables. Just check that each of listed tables actually matches the pattern from query.

Reply:
I like that